### PR TITLE
Remove all ensure_root and replace with EnsureOrigin<> #2

### PR DESCRIPTION
**Pull request by herou**
**Summary of changes**
Changes introduced in this pull request:
- Replace ensure_root with ensure_origin 

**Reference issue to close (if applicable)**

https://github.com/webb-tools/dkg-substrate/issues/441

### CODE REVIEW

**Comment by herou** [comment]
@dutterbutter @drewstone Can you please review this?

**Comment by drewstone** [comment]
Can you fix the lint?

**Comment by drewstone** [comment]
Also why are the types all called AssetModifierOrigin when some pallets have nothing to do with assets?

File: pallets/dkg-metadata/src/lib.rs [hunk 1]
```diff
@@ -216,6 +216,8 @@ pub mod pallet {
 		/// A type that gives allows the pallet access to the session progress
 		type NextSessionRotation: EstimateNextSessionRotation<Self::BlockNumber>;
 
+		type AssetModifierOrigin: EnsureOrigin<Self::RuntimeOrigin>;
```
Comment (drewstone) [inline pallets/dkg-metadata/src/lib.rs:219]: I do not agree with the naming convention used.

Comment (drewstone) [reply pallets/dkg-metadata/src/lib.rs:219]: Needs to be relevant for each pallet.

**Review by drewstone — CHANGES_REQUESTED** [review]

**Comment by drewstone** [comment]
Is this still active @herou? I will close next week without comment.

**Comment by drewstone** [comment]
Closing this since it's gone inactive.

**Comment by drewstone** [comment]
Reopened @herou

**Pull request closed without merging**